index.html

```diff
@@ -4,9 +4,50 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Wanderegionen in Deutschland</title>
 </head>
 <body>
-    
+    <header>
+        <img src="https://www.ndr.de/ratgeber/reise/hamburg/harburgerberge110_v-contentxl.jpg" alt="">
+    </header>
+    <main>
+        <h2>Die Alpen</h2>
+        <p>Die Alpen sind das höchste Hochgebirge im Inneren Europas.</p>
+        <h3>die schönsten Gipfel der Alpen</h3>
+        <ul>
+            <li>Zugspitze</li>
+            <li>Mont Blanc</li>
+            <li>Matterhorn</li>
+        </ul>
+        <h2>Das Erzgebirge</h2>
+        <p>Das Erzgebirge ist ein Mittelgebirge in Sachsen und Böhmen.</p>
+        <h3>die schönsten Gipfel des Erzgebirges</h3>
+        <ul>
+            <li>Großer Fichtelberg</li>
+            <li>Auersberg</li>
+            <li>Katzenstein</li>
+        </ul>
+        <h2>Der Schwarzwald</h2>
+        <p>Der Schwarzwald ist Deutschlands höchstes und größtes zusammenhängendes Mittelgebirge und liegt im Südwesten Baden-Württembergs.</p>
+        <h3>die schönsten Gipfel des Schwarzwaldes</h3>
+        <ul>
+            <li>Schauinsland</li>
+            <li>Kandel</li>
+            <li>Belchenxy</li>
+        </ul>
+    </main>                           
+    <footer>
+        <h4>Quellen: </h4>
+        <nav>
```

Comment:
Is the nav element appropriate when listing the sources? Maybe an aside element within the main element?